preprocessing-service/masker.py

```diff
@@ -59,6 +70,10 @@ def mask(self, content: str):
 
 
 masking_list = [
+    {
+        "regex_pattern": "[^\\s]+.go : [0-9]+",
+        "mask_with": "GO_FILE_PATH"
+    },
     {
         "regex_pattern": "[a-z0-9]+[\\._]?[a-z0-9]+[@]\\w+[.]\\w{2,3}",
         "mask_with": "EMAIL_ADDRESS",
```

Comment:
Don't think email addresses appear in control plane logs. This mask as well as any others that we don't anticipate to appear in control plane should be commented out